pyproject.toml

```diff
@@ -22,5 +112,87 @@ filterwarnings = [
     # test_config_jupytext_jupyter_fs_meta_manager
     'ignore:The alias `_\(\)` will be deprecated. Use `_i18n\(\)` instead.:FutureWarning',
     # Conda/Python 3.7/Windows - ImportError while loading conftest
-    "ignore:the imp module is deprecated in favour of importlib:DeprecationWarning"
+    "ignore:the imp module is deprecated in favour of importlib:DeprecationWarning",
+    "ignore:Code cell is missing an id field:FutureWarning",
+    "ignore:There is no current event loop:DeprecationWarning",
+]
+
+[tool.coverage.run]
+omit = [
+    "tests/*",
+]
+parallel = true
+branch = false
+
+[tool.coverage.report]
+# Regexes for lines to exclude from consideration
+exclude_lines = [
+    "if self.debug:",
+    # Have to re-enable the standard pragma
+    "pragma: no cover",
+
+    "if __name__ == .__main__.:",
+
+    # Don't complain if tests don't hit defensive assertion code:
+    "raise NotImplementedError",
+    "except ImportError",
+]
+ignore_errors = true
+omit = [
+    "tests/*",
+    "jupytext/version.py",
+    "*/site-packages/*"
+]
+
+# To use tox, see https://tox.readthedocs.io
+# Simply pip or conda install tox
+# If you use conda, you may also want to install tox-conda
+# then run `tox` or `tox -- {pytest args}`
+# To run in parallel using `tox -p` (this does not appear to work for this repo)
+
+# To rebuild the tox environment, for example when dependencies change, use
+# `tox -r`
+
+# Note: if the following error is encountered: `ImportError while loading conftest`
+# then then deleting compiled files has been found to fix it: `find . -name \*.pyc -delete`
+[tool.tox]
+legacy_tox_ini = """
+    [tox]
+    envlist = py{35,36,37,38}
+
+    [testenv:py{35,36,37,38}]
+    ; recreate = false
+    deps =
+        -rrequirements.txt
+        -rrequirements-dev.txt
+        jupyter-fs
+    commands_pre = python -m ipykernel install --name jupytext-tox-{envname} --user
+    commands = pytest {posargs}
+
+    [testenv:docs-{update,clean}]
+    deps = -rdocs/doc-requirements.txt
+    whitelist_externals = rm
+    commands =
+        clean: rm -rf docs/_build
+        sphinx-build {posargs} -nW --keep-going -b html docs/ docs/_build/html
+"""
+
+# Jupyter releaser config for check-release.yml workflow
+[tool.jupyter-releaser]
+skip = [
+    "bump-version",
+]
```

Comment:
What are the steps done in `juptyer-releaser`?

Reply:
Quite a few of them actually. [Docs](https://jupyter-releaser.readthedocs.io/en/latest/reference/releaser_cli.html) have references. To see it in action, look at [logs here](https://github.com/mahendrapaipuri/jupyter-power-usage/actions/runs/6495722604/job/17641223614)